assets/environment-interpreter/activate/activate

```diff
@@ -265,6 +258,18 @@ case "$_flox_shell" in
     ;;
 esac
 
+if [ "$_FLOX_ENV_ACTIVATION_MODE" = "build" ] || [ "$_FLOX_ENV_ACTIVATION_MODE" = "build-sandbox" ]; then
+  if [ ! $# -gt 0 ]; then
+    echo "build mode is only supported for running a command" >&2
+    exit 1
+  fi
+  # We want to run hooks without a state dir, since we'll never attach to this
+  # activation
+  start ""
```

Comment:
Ok, I see the expected pattern now. The wording in the other PR makes it sound like the function could be called without arguments, whereas this is passing an empty string. I would call this out more explicitly that you expect the string to be empty to avoid confusion (and bugs).